frontend/src/components/LegalNotice/legalNotice.cssmodule.scss

```diff
@@ -0,0 +1,30 @@
+body {
```

Comment:
You should always use multiple of 8 in your size

Reply:
and don't override the body !!!!

Reply:
Multiple of 8, i tried but it's too big or too small. To look like the exemple it's between.

Reply:
I think that if i use body it's only for the legalNotice page in this case, but it's for all the body ?

Reply:
I put .container instead of body but i didn't touched the size. Waiting for an answer.